glib/sys/Cargo.toml

```diff
@@ -93,7 +94,10 @@ version = "2.76"
 version = "2.77"
 
 [package.metadata.system-deps.glib_2_0.v2_80]
-version = "2.79"
+version = "2.80"
```

Comment:
```
  > PKG_CONFIG_ALLOW_SYSTEM_CFLAGS=1 C:\gtk-build\gtk\x64\release\bin\pkgconf.exe --libs --cflags glib-2.0 glib-2.0 >= 2.80

  The system library `glib-2.0` required by crate `glib-sys` was not found.
  The file `glib-2.0.pc` needs to be installed and the PKG_CONFIG_PATH environment variable must contain its parent directory.
  The PKG_CONFIG_PATH environment variable is not set.
```

It looks like Windows currently uses GLib 2.79.X? How do we update that @bilelmoussaoui ? :)

Reply:
@pbor is the windows expert in this repo, i only know how to deal with msvc

Reply:
Maybe like this. Let's see

Reply:
Yes